website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -64,37 +64,38 @@ Properties without a default value are required.
+| `data.attributes.assessments-enabled`           | boolean | `false`   | (previously `drift-detection`) Whether or not Terraform Cloud performs health assessments for the workspace. May be overriden by the organization setting `assessments-enforced`. Only available for Plus tier organizations, in workspaces running Terraform version 0.15.4+ and operating in [Remote execution mode](/terraform/cloud-docs/workspaces/settings#execution-mode).                                                                                                                               |
+| `data.attributes.auto-apply`                    | boolean | `false`   | Whether to automatically apply changes when a Terraform plan is successful in runs initiated by VCS, UI or CLI, [with some exceptions](/terraform/cloud-docs/workspaces/settings#auto-apply-and-manual-apply).                                                                                                                                                                                                                                                                                                  |
+| `data.attributes.auto-apply-run-trigger`        | boolean | `false`   | Whether to automatically apply changes when a Terraform plan is successful in runs initiated by run triggers. Not supported in Terraform Enterprise.                                                                                                                                                                                                                                                                                                   |
+| `data.attributes.auto-destroy-at`               | string  | (nothing) | Timestamp when the next scheduled destroy run will occur, refer to [Scheduled Destroy](/terraform/cloud-docs/workspaces/settings/deletion#automatically-destroy).                                                                                                                                                                                                                                                                                                                                               |
+| `data.attributes.auto-destroy-activity-duration`| string  | (nothing) | Value and units for [automatically scheduled destroy runs based on workspace activity](/terraform/cloud-docs/workspaces/settings/deletion#automatically-destroy). Values must be greater than 0 and valid units are `d` and `h`. For example, to set to queue destroy runs after fourteen days of inactivity set `auto-destroy-activity-duration: "14d"`.                                                                                                                                                       |
```

Comment:
is it worth documenting upper limit of 9999?

Reply:
I think so, I think it's always good to document those kinds of edge cases.

Reply:
Updated the description to be
```md
Value and units for [automatically scheduled destroy runs based on workspace activity](/terraform/cloud-docs/workspaces/settings/deletion#automatically-destroy). Valid values be greater than 0 and four digits or less. Valid units are `d` and `h`. For example, to queue destroy runs after fourteen days of inactivity set `auto-destroy-activity-duration: "14d"`.
```

Let me know if there's a clearer way of saying that.

Reply:
Small tweak to your above language, but I think this looks good!
```suggestion
Value and units for [automatically scheduled destroy runs based on workspace activity](/terraform/cloud-docs/workspaces/settings/deletion#automatically-destroy). Valid values are greater than 0 and four digits or less. Valid units are `d` and `h`. For example, to queue destroy runs after fourteen days of inactivity set `auto-destroy-activity-duration: "14d"`.
```